hasher-matcher-actioner/webapp/src/Signals.jsx

```diff
@@ -15,7 +15,9 @@ export default function Signals() {
 
   useEffect(() => {
     fetchSignalSummary().then(summaries => {
-      setSignalSummary(summaries.signals);
+      if (summaries.signals.length) {
```

Comment:
Dunno if I'm reading this right, but if this check prevents the call to `setSignalSummary()` then the spinner will show forever, even though the API call completed, right? So... introducing this maybe you need to let `setSignalSummary()` unconditionally and then have two `<Collapse>` elements, one for data and one for a "no data returned" kind of message?

Reply:
yeah, going from breaking -> spinning with this change, will add TODO comment for now.